src/spatialdata_plot/pl/utils.py

```diff
@@ -10,11 +10,11 @@
 from typing import Any, Literal, Union
 
 import matplotlib
+import matplotlib.patches
 import matplotlib.patches as mpatches
 import matplotlib.patches as mplp
```

Comment:
Any particular reason why we need all 3 lines?

Reply:
No 🥸